iree/compiler/InputConversion/MHLO/MHLOToLinalgOnTensors.cpp

```diff
@@ -207,7 +203,101 @@ struct FftOpConversion : public OpConversionPattern<mhlo::FftOp> {
     return success();
   }
 };
-}  // namespace
+
+/// Convert mhlo.constant op into std.const.
+struct ConstOpConversion : public OpConversionPattern<mhlo::ConstOp> {
```

Comment:
This should already be in the upstream patterns: https://github.com/tensorflow/tensorflow/blob/a3b902d05248c95600b05a7188cbb7a61594a2ae/tensorflow/compiler/mlir/hlo/lib/Dialect/mhlo/transforms/legalize_to_linalg.cc#L1395

Reply:
May be - I just moved it because the file was organized badly.

Reply:
Removed.